src/schema/artwork/contextGrids/ArtistArtworkGrid.ts

```diff
@@ -0,0 +1,61 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import {
+  ContextGridType,
+  formDefaultGravityArgs,
+} from "schema/artwork/contextGrids"
+import { artworkConnection } from "schema/artwork"
+import { connectionFromArraySlice } from "graphql-relay"
+import { artistArtworkArrayLength } from "schema/artist"
+import { pageable } from "relay-cursor-paging"
+
+export const ArtistArtworkGridType = new GraphQLObjectType<
+  { artist: any; artwork: any },
+  any
+>({
+  name: "ArtistArtworkGrid",
+  interfaces: [ContextGridType],
+  fields: () => ({
+    title: {
+      type: GraphQLString,
+      resolve: ({ artist }) => {
+        if (!artist) return null
+        return `Other works by ${artist.name}`
+      },
+    },
+    ctaTitle: {
+      type: GraphQLString,
+      resolve: ({ artist }) => {
+        if (!artist) return null
+        return `View all works by ${artist.name}`
+      },
+    },
+    ctaDestination: {
```

Comment:
#minor should this be `ctaHref`? my hypermedia mind is asking.

Reply:
Ooo probably.